extras/gke-autopilot/README.md

```diff
@@ -0,0 +1,176 @@
+# StatefulSet worload with HPA
```

Comment:
Is this more-or-less the same tutorial than the one we want to publish? If so, I'd opt for _not_ having this copy here, and instead shorten the README to a few lines that explains what these files are for, very briefly

Reply:
I'd also think of a different name than `gke-autopilot` (the content is not inherently Autopilot-based or _about_ Autopilot) -- we could name it something to signal that it's "hpa postgres" or something along those lines, and we may want to tweak the directory structure a bit. Think of it like this: the files / directories in the Bank of Anthos repo should be platform and especially tutorial agnostic as much as possible

Reply:
Done. README file should be updated with a link to the tutorial when it will be published.